commons-impl/src/main/java/com/connexta/transformation/commons/inmemory/InMemoryTransformationManager.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) Connexta
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package com.connexta.transformation.commons.inmemory;
```

Comment:
I wasn't sure if I should put all of these classes in their own "in-memory" module or if they should just be in their own package. Open to feedback on this.

Reply:
both

Reply:
I don't see any reason why they need to be in their own module. 🤷‍♂